tests/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ else()
 endif()
 
 string(REGEX MATCH
-  "([0-9])\\.([0-9])\\.([0-9])" GTEST_VERSION_PARSED
+  "([0-9])\\.([0-9]+)\\.([0-9])" GTEST_VERSION_PARSED
```

Comment:
Any reason to add multi-digit support only for the minor version and not major or patch?

Reply:
not really